ios/Mattermost.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,24 +1,6 @@
 {
   "object": {
     "pins": [
-      {
-        "package": "OpenGraph",
-        "repositoryURL": "https://github.com/satoshi-takano/OpenGraph.git",
-        "state": {
-          "branch": null,
-          "revision": "382972f1963580eeabafd88ad012e66576b4d213",
-          "version": "1.4.1"
-        }
-      },
-      {
-        "package": "Sentry",
-        "repositoryURL": "https://github.com/getsentry/sentry-cocoa.git",
-        "state": {
-          "branch": "8.15.2",
-          "revision": "c60f232c96264c3bea75bc0b81e6126b2f7475ee",
-          "version": null
-        }
-      },
```

Comment:
this does not seem to be on purpose

Reply:
Updating the version touched the right project file places for this to never happen again, hopefully...